src/Microsoft.Health.Dicom.Core/Exceptions/DataPartitionsFeatureCannotBeDisabledException.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Globalization;
+
+namespace Microsoft.Health.Dicom.Core.Exceptions
+{
+    /// <summary>
+    /// Exception that is thrown when data partitions feature is disabled.
+    /// </summary>
+    public class DataPartitionsFeatureCannotBeDisabledException : BadRequestException
```

Comment:
Is this really a `BadRequest`? It seems more general purpose as this is thrown at the server start-up, like an `InvalidOperationException`?

Reply:
Yes makes sense.